src/apps/recommendation/recommendation.tsx

```diff
@@ -0,0 +1,62 @@
+import clsx from "clsx";
+import * as React from "react";
+import Arrow from "../../components/atoms/icons/arrow/arrow";
+import Link from "../../components/atoms/links/Link";
+import { useGetMaterialQuery } from "../../core/dbc-gateway/generated/graphql";
+import { constructMaterialUrl } from "../../core/utils/helpers/url";
+import { Work } from "../../core/utils/types/entities";
+import { WorkId } from "../../core/utils/types/ids";
+import { useUrls } from "../../core/utils/url";
+import RecommendationMaterialSkeleton from "./RecommendationSkeleton";
+import RecommendedMaterial from "../recommended-material/RecommendedMaterial";
+
+export type RecommendationProps = {
+  positionImageRight?: boolean;
+  wid: WorkId;
+};
+
+const Recommendation: React.FC<RecommendationProps> = ({
+  positionImageRight,
+  wid
+}) => {
+  const u = useUrls();
+  const materialUrl = u("materialUrl");
+
+  const { data, isLoading } = useGetMaterialQuery({
+    wid
+  });
+
+  if (isLoading || !data?.work) {
+    return <RecommendationMaterialSkeleton />;
+  }
+
+  const {
+    work: {
+      titles: { full: materialFullTitle },
+      abstract
+    }
+  } = data as { work: Work };
```

Comment:
Please reconsider using `as` here.

As far as as I can tell it should not be necessary. In general we want to avoid using `as`.